### PR TITLE
[forge] load vs perf benchmark

### PR DESCRIPTION
**Pull request by igor-aptos**
### Test Plan

- - -
This change is [Reviewable](https://reviewable.io/reviews/aptos-labs/aptos-core/3945)

### CODE REVIEW

**Review by zekun000 — APPROVED** [review]
the test name doesn't seem very intuitive to me

**Comment by igor-aptos** [comment]
@zekun000 any suggestion for a better name? :)

File: testsuite/testcases/src/load_vs_perf_benchmark.rs [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+            tps,
+            stats,
+            ledger_transactions,
+            actual_duration,
+        })
+    }
+}
+
+impl NetworkTest for LoadVsPerfBenchmark {
+    fn run<'t>(&self, ctx: &mut NetworkContext<'t>) -> Result<()> {
```
Comment (sitalkedia) [inline testsuite/testcases/src/load_vs_perf_benchmark.rs:59]: Do you want to run this test in continuous mode or is it just for adhoc testing?

Comment (igor-aptos) [reply testsuite/testcases/src/load_vs_perf_benchmark.rs:59]: At the moment adhoc.

If I think of success criteria for such a great,  we can make it run on continuous too.

**Review by sitalkedia — APPROVED** [review]

**Pull request merged**